src/transformers/models/mistral/modeling_mistral.py

```diff
@@ -344,15 +334,16 @@ def forward(
         past_key_value: Optional[Cache] = None,
         output_attentions: bool = False,
         use_cache: bool = False,
-        **kwargs,
+        cache_position: Optional[torch.LongTensor] = None,
```

Comment:
same here should be copied from Gemma or llama no?

Reply:
I think flash attention has to take care of sliding window logic itself

Reply:
yeah, you're right, it's mostly passing the sliding window arg to the kernel